src/Analyzers/CSharp/Tests/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -4731,13 +4732,18 @@ interface IGoo
 
             public class C : IGoo
             {
-                void IGoo.Goo(DateTime x)
+                void IGoo.Goo([DateTimeConstant(100), Optional] DateTime x)
                 {
                     throw new NotImplementedException();
                 }
             }
             """,
-codeAction: ("True;False;False:global::IGoo;Microsoft.CodeAnalysis.ImplementInterface.AbstractImplementInterfaceService+ImplementInterfaceCodeAction;", 1));
+            Options = { AllOptionsOff },
+
+            CodeActionEquivalenceKey = "True;False;False:global::IGoo;Microsoft.CodeAnalysis.ImplementInterface.AbstractImplementInterfaceService+ImplementInterfaceCodeAction;",
+            // 🐛 one value is generated with 0L instead of 0
```

Comment:
not sure what this means.

Reply:
See also `TestIntegralAndFloatLiterals`; when specifying `DateTimeConstant(100)`, the semantic structure equivalence check does not forgive the mismatching literal types, despite the available conversion. Removing the CodeActionValidationMode causes the test to fail saying that `"100" does not equal "100"`, where the one is int and the other is long. Apparently the generated argument is perceived as being `int` when the attribute's argument expects a long. I have not dug up the real cause for this problem, so I just copied the note from the other test to indicate the problem.

Reply:
The provider no longer emits attributes, so this was removed.